cmd/kots/cli/admin-console-copy-public-images.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/kotsadm"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func AdminCopyPublicImagesCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "copy-public-images [registry host]",
+		Short:         "Copy public admin console images",
+		Long:          "Copy public admin console images to a private registry",
+		Hidden:        true,
+		SilenceUsage:  true,
+		SilenceErrors: false,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				cmd.Help()
+				os.Exit(1)
+			}
+
+			endpoint := args[0]
+			options, err := genAndCheckPushOptions(endpoint, cmd)
+			if err != nil {
+				return err
+			}
+
+			v := viper.GetViper()
+			namespace, err := getNamespaceOrDefault(v.GetString("namespace"))
+			if err != nil {
+				return errors.Wrap(err, "failed to get namespace")
+			}
+
+			err = kotsadm.CopyImages(*options, namespace)
+			if err != nil {
+				return errors.Wrap(err, "failed to copy images")
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().String("registry-username", "", "user name to use to authenticate with the registry")
+	cmd.Flags().String("registry-password", "", "password to use to authenticate with the registry")
+	cmd.Flags().Bool("skip-registry-check", false, "skip the connectivity test and validation of the provided registry information")
+
+	cmd.Flags().String("source-registry", "docker.io", "source registry for public images")
```

Comment:
where is this used?

Reply:
argh, I'm trying to get better at removing these code bread crumbs 🤦. Before removing the `sourceEndpoint` parameter from `CopyImages`, I was using this to default it to "docker.io", but that's not needed now